dotnet-desktop-guide/framework/winforms/controls/how-to-create-an-unbound-windows-forms-datagridview-control.md

```diff
@@ -15,9 +13,9 @@ ms.assetid: b5d4b47d-9a28-4d88-9dba-0a3c90fba71d
 ---
 # How to: Create an Unbound Windows Forms DataGridView Control
 
-The following code example demonstrates how to populate a <xref:System.Windows.Forms.DataGridView> control programmatically without binding it to a data source. This is useful when you have a small amount of data that you want to display in a table format.
+The following code example demonstrates how to populate a <xref:System.Windows.Forms.DataGridView> control programmatically without binding it to a data source. This is useful when you have a small amount of data that you want to display in a table format.  
```

Comment:
It looks like you added a space to the end of some sentences throughout this update. Please remove the spaces you added (unless that messes up the formatting of the following sections). Any superfluous spaces result in more memory/processing/handling of the files.

Reply:
Added empty lines. 
MD022 - Headings should be surrounded by blank lines